test/test-functions

```diff
@@ -99,11 +103,18 @@ run_qemu() {
         exit 1
     fi
 
-    KERNEL_APPEND="root=/dev/sda1 \
+if [[ "$LOOKS_LIKE_SUSE" ]]; then
+    PARAMS+="systemd.mask=systemd-networkd.service "
+    PARAMS+="rd.hostonly=0"
```

Comment:
I think this one might be used when the tests are run on Fedora. However, that just works around one problem with a host-only initramfs. To really test `systemd` built from the source, a custom initramfs should be generated as was mentioned in https://github.com/systemd/systemd/issues/4607.